### PR TITLE
[Downgrade PHP 7.1] Fixes #4196 Add DowngradePipeToMultiCatchExceptionRector

### LINKED ISSUE #4196: [Downgrade PHP 7.1] Multi catch exception handling
# Feature Request

Downgrade new feature from PHP 7.1 to its equivalent PHP 7.0 code. [Feature description](https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.mulit-catch-exception-handling)
 
## Diff 

```diff
try {
    // some code
-} catch (FirstException | SecondException $e) {
+} catch (\Exception $e) {
    // handle first and second exceptions
}
```

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+use Iterator;
+use Rector\DowngradePhp71\Rector\TryCatch\DowngradePipeToMultiCatchExceptionRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class DowngradePipeToMultiCatchExceptionRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
```
Comment (TomasVotruba) [inline rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php:16]: Downgrade rules tests can run only on PHP they're downgrading and hire. So there is no syntax error:

```suggestion
    /**
     * @requires PHP 8.0
     * @dataProvider provideData()
     */
```

Comment (TomasVotruba) [reply rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php]: The rest look good, so I'm merge this :+1:

File: rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+use Iterator;
+use Rector\DowngradePhp71\Rector\TryCatch\DowngradePipeToMultiCatchExceptionRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class DowngradePipeToMultiCatchExceptionRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @requires PHP 8.0
```
Comment (TomasVotruba) [inline rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php:15]: Wrong version :) 

```suggestion
     * @requires PHP 7.1
```

**Pull request merged**